src/file.js

```diff
@@ -35,11 +35,15 @@ function normalizePath(str) {
 
 /**
  * Check whether a resource is external or not
- * @param {string} href Path
+ * @param {string|Vinyl} href Path
```

Comment:
Not super familiar with jsdoc so please confirm this is right

Reply:
Shouldn't be ;)
Do you have a hint to start debugging where the Vinyl object is coming from?

Reply:
It happens in one of the vinyl tests. I'll revert this part so that we get more info.